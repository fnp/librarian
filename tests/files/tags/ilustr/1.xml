<ilustr src="https://redakcja.wolnelektury.pl/media/images/kipling-takie-sobie-bajeczki-calosc/kot.jpg" alt="Kot" />
