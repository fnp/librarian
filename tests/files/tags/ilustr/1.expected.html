<div id="book-text"><img class="ilustr" alt="Kot" title="Kot" src="https://redakcja.wolnelektury.pl/media/images/kipling-takie-sobie-bajeczki-calosc/kot.jpg"></div>
